node/Makefile

```diff
@@ -231,12 +230,27 @@ CGO_LDFLAGS=""
 CGO_CFLAGS=""
 endif
 
+ifeq ($(FIPS), true)
```

Comment:
Is this clause a little low down in the file for this to work? I figured that it would need to be set before it is first called in the Makefile in order for it to guarantee that there's a definition?